Add eyw-symbol (reverse star symbol) for ootransformers

Dr4UX opened the pull request:
I just starting to work with git, so I accidentally deleted my last pr. This one should now be less confusing. Here my original pr:

I often see the Y-symbol in transformers turned around (with the vertical line upwards) and prefer it that way. I guess the easiest way is to just add another symbol for the ootransformers with the 180°-rotated Y-symbol. I hope you like it.

![image](https://github.com/circuitikz/circuitikz/assets/86589394/61f47f71-446a-47af-863b-131c0f68134c)

Rmano commented:
In general I like it. Please fix the errors (you can force-push this branch, no problem). When it's done I'll merge.

Dr4UX commented:
I searched for a while now for the error and I have no more ideas where else to look. To create the `eyw`-Symbol I just copied everything in the `tex/pgfcircbipoles.tex` that had a `wye` and changed it to `eyw`. Then I added the definition of the `eyw`-symbol. If I didn't forget anything I have no more ideas what could be wrong.

Rmano commented:
> I searched for a while now for the error and I have no more ideas where else to look. To create the `eyw`-Symbol I just copied everything in the `tex/pgfcircbipoles.tex` that had a `wye` and changed it to `eyw`. Then I added the definition of the `eyw`-symbol. If I didn't forget anything I have no more ideas what could be wrong.

The error seems to be there:
```
(./circuitikzmanual.tmp

! Undefined control sequence.

\pgf@circ@drawsmartlabels ...m \pgfcircmathresult 

                                                  > 20 \ifnum \pgfcircmathre...

l.2 ...urcetrans, prim=zig, sec=delta, o-] ++(2,0)

                                                  

Output written on circuitikzmanual.pdf (66 pages).
```

...so probably there is something wrong in the `\if... \else...\fi` chain  that you modifyed there.  The error comes out only when adding documentation, because is when the code is exercised (this is why I normally prefer just one commit per function, including both code and docs). It seems that you added several `\else` without adding `\fi`s, which is basically impossible: in TeX `\if ... \else ... \else ... \fi` does not work, you have *only* `\if  ... \else ... \fi`

Rmano commented:
Try to add a fourth `\fi` at the end of the blocks where you added the `\else` (There is a `\fi\fi\fi`, add a `\fi` there). I thought I could change it here but it wants me to create another branch... `git` sometimes is a bit mysterious.

Dr4UX commented:
Thanks! I fixed it. Everything should work now just fine.

Rmano commented:
Ok, will squash and merge. Thanks for the contribution!
